### DISCUSSION TITLE
Cannot upload CSV 7KB files in the backend, self hosted

### DISCUSSION THREAD

**Discussion by sghriyadh** [Budibase/budibase#11509] [Help] [answered]
Hi
i have a self hosted budibase on dockers.
I have 7000KB CSV with 95,000 rows

It is giving error file is too large.

Is CSV upload, restricted by rows numbers or file size ?

**Comment by ConorWebb96** [comment]
Hey @sghriyadh,

It would be file size and not the number of rows. Try splitting your CSV up into 2 parts and upload both CSVs separately.

I hope this helps.

**Reply by sghriyadh** [reply]
File is only 7MB . this is well below 20MB limit.

Should i file it as a bug ?

**Reply by ConorWebb96** [reply, marked as answer]
Hey @sghriyadh,

The upload limit for CSV is 5mbs. Sadly this isn't currently configurable. I'd recommend splitting up the CSV into 2 parts and import each half separately.

You can definitely create a feature request if you wish for this to be configurable for self-host users, you can do this here; https://github.com/Budibase/budibase/issues/new/choose

**Comment by Freundschaft** [comment]
does the same limit apply to JSON imports?

**Reply by ConorWebb96** [reply]
Yes, most likely.
